apps/sync/manifest.webapp

```diff
@@ -8,6 +8,9 @@
      "name": "The Gaia Team",
      "url": "https://github.com/mozilla-b2g/gaia"
   },
+  "permissions": {
+    "settings": { "access": "readwrite" }
```

Comment:
please add `"description": "Necessary to see if logging is enabled"` and change to `readonly` as discussed on irc